_studio/shared/asc/include/asc_common_impl.h

```diff
@@ -178,7 +178,7 @@ static inline __m256i LoadPartialYmm(unsigned char *pSrc, mfxI32 len)
 #pragma warning(disable:4505)
 #endif
 
-static void calc_RACA_4x4_C(mfxU8 *pSrc, mfxI32 pitch, mfxI32 *RS, mfxI32 *CS) {
+static inline void calc_RACA_4x4_C(mfxU8 *pSrc, mfxI32 pitch, mfxI32 *RS, mfxI32 *CS) {
```

Comment:
This place and file in general looks STRANGE (because of _missing_ inline and presence of the warning disable above). If author did not want to make this function inline, then he should have defined it in the .cpp file. If he wanted make it inline, then why disable a warning?

Actually the whole this file looks strange to me. Functions defined in it are not that small actually. I am not sure they should be made inline at the first place. @spossos: can you, please, comment. @onabiull: what do you think?